lib/resources/order-recipients.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Orders Recipients resources
+ *
+ * @module resources/orders-recipients
+ * @copyright 2015–2020 RewardOps Inc.
+ */
+const axios = require('axios');
+const { get } = require('lodash');
+
+const config = require('../config');
+const { storeOrderRecipientSchema } = require('../schemas/store-order-recipient');
+
+const debugLogs = [];
+
+const setV5Token = async () => {
+  const {
+    data: { access_token: accessToken },
+  } = await axios.post(
+    `${config.get('piiServerUrl')}/api/v5/auth/token`,
+    {
+      grant_type: 'client_credentials',
+    },
+    {
+      auth: {
+        username: config.get('clientId'),
+        password: config.get('clientSecret'),
+      },
+    }
+  );
+  debugLogs.push({ function: 'storeOrderRecipient.getV5Token', accessToken });
+  axios.defaults.headers.common.Authorization = `Bearer ${accessToken}`;
+};
+
+const createOrder = ({ contextId }) => async (params, callback) => {
+  debugLogs.push({ function: 'createOrder', params });
+  try {
+    const response = await axios.post(`${config.get('piiServerUrl')}/api/v4/programs/${contextId}/orders`, params);
+    callback(null, response.data, response);
+  } catch (error) {
+    callback({ error, debugLogs });
+  }
+};
+
+/**
+ * Higher order function for creating a `create` order function in context.
+ *
+ * NOTE: This is a v5 API endpoint, therefore ignores the `apiVersion` config prop
+ *
+ * @param {module:resources/orders~OrderContext} orderContext Order context object
```

Comment:
`OrderContext` doesn't match the signature used here. Let's create a `V5OrderContext` definition

Reply:
done